pkg/blocktx/client.go

```diff
@@ -10,10 +10,10 @@ import (
 )
 
 // check if Client implements all necessary interfaces
-var _ BlocktxClient = &Client{}
+var _ BlockClient = &Client{}
 var _ MerkleRootsVerifier = &Client{}
 
-type BlocktxClient interface {
+type BlockClient interface {
```

Comment:
This could be renamed to `Watcher` or something like that, because this interface is used only in k8s_watcher and calling it `BlockClient` instead of `BlocktxClient` seems like a dirty hack 😄

Reply:
Done